packages/blockchain-api/src/schema.ts

```diff
@@ -106,26 +217,57 @@ export const typeDefs = gql`
       offset: Int
     ): [Transfer]
 
+    transactions(
+      address: Address!
+      token: Token!
+      localCurrencyCode: String
+      # pagination
+      before: String
+      last: Int
+      after: String
+      first: Int
+    ): TransactionConnection
+
     currencyConversion(
       sourceCurrencyCode: String
       currencyCode: String!
-      timestamp: Float
+      timestamp: Timestamp
     ): ExchangeRate
   }
 `
 
 interface Context {
   dataSources: DataSources
+  localCurrencyCode?: string
 }
 
 export const resolvers = {
   Query: {
-    events: async (_source: any, args: EventArgs, { dataSources }: Context) => {
+    events: async (_source: any, args: EventArgs, context: Context) => {
+      const { dataSources } = context
       return dataSources.blockscoutAPI.getFeedEvents(args)
     },
     rewards: async (_source: any, args: EventArgs, { dataSources }: Context) => {
       return dataSources.blockscoutAPI.getFeedRewards(args)
     },
+    transactions: async (_source: any, args: TransactionArgs, context: Context) => {
```

Comment:
My problem with `transaction` is that it's too vague. It could be anything from an ERC20 token transfer to a verification request to a smart contract deployment. Is there a more descriptive name we could use?

Reply:
Makes sense, `tokenTransactions` feels like the right name then, what do you think?

Reply:
Sounds good

Reply:
done 👍